db/migrations/20240623020146_AddSessionLoggedOutAt.up.sql

```diff
@@ -0,0 +1,3 @@
+-- Add migration script here
+ALTER TABLE Sessions
+ADD COLUMN logged_out_at TIMESTAMPTZ NULL;
```

Comment:
We also want to record when you logout of a session, so that we can treat it as not-active anymore